pandas-stubs/io/excel/_base.pyi

```diff
@@ -9,6 +9,7 @@ from types import TracebackType
 from typing import (
     Any,
     Literal,
+    TypeAlias,
```

Comment:
You'll need to import `TypeAlias` from `typing_extensions` so we can support earlier versions of python.